### PR TITLE
MFE-4 table update flow

### PR DESCRIPTION
**Pull request by gidesan**

### CODE REVIEW

File: react/form/src/components/ConferenceEditFormContainer.js [hunk 1]
```diff
@@ -22,33 +21,43 @@ class ConferenceEditFormContainer extends PureComponent {
   }
 
   async componentDidMount() {
+    this.fetchConference();
+  }
+
+  componentDidUpdate(prevProps) {
```
Comment (rimselis) [inline react/form/src/components/ConferenceEditFormContainer.js:26]: `async`?

File: react/form/src/components/ConferenceForm.js [hunk 2]
```diff
@@ -23,93 +24,108 @@ const styles = theme => ({
...
-              className={classes.textField}
-              onChange={handleChange}
-              onBlur={handleBlur}
-              value={values.name}
-              name="name"
-              label={t('entities.conference.name')}
-            />
+class ConferenceForm extends PureComponent {
+  // eslint-disable-next-line react/static-property-placement
+  static contextType = MountPointContext;
```
Comment (rimselis) [inline react/form/src/components/ConferenceForm.js]:
```
class ConferenceForm extends PureComponent {
    // ...
}
ConferenceForm.contextType = MountPointContext;
```

File: react/form/src/components/ConferenceForm.js [hunk 3]
```diff
@@ -1,18 +1,19 @@
 import 'date-fns';
 import dateFnsLocales from 'i18n/dateFnsLocales';
 import DateFnsUtils from '@date-io/date-fns';
-import React from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { formValues, formTouched, formErrors } from 'components/__types__/conferenceTypes';
 import { withFormik } from 'formik';
 import { withTranslation } from 'react-i18next';
```
Comment (rimselis) [inline react/form/src/components/ConferenceForm.js:8]: `import i18next from 'i18next';`

and `i18next.language` / `i18next.t()`

File: react/form/src/components/ConferenceAddFormContainer.js [hunk 4]
```diff
@@ -21,19 +21,17 @@ class ConferenceAddFormContainer extends PureComponent {
...
+    const { t, onCreate } = this.props;
     try {
       const createdConference = await apiConferencePost(conference);
-      const { onCreate } = this.props;
       onCreate(createdConference);
-
       this.setState({
         notificationMessage: t('common.dataSaved'),
-        notificationStatus: 'success',
+        notificationStatus: Notification.SUCCESS,
```
Comment (rimselis) [inline react/form/src/components/ConferenceAddFormContainer.js:33]: `notificationStatus` not in the initial state

File: react/form/src/components/ConferenceEditFormContainer.js (hunk 1 above)
Comment (gidesan) [reply react/form/src/components/ConferenceEditFormContainer.js:26]: fixed

File: react/form/src/components/ConferenceForm.js (hunk 2 above)
Comment (gidesan) [reply react/form/src/components/ConferenceForm.js]: fixed

File: react/form/src/components/ConferenceAddFormContainer.js (hunk 4 above)
Comment (gidesan) [reply react/form/src/components/ConferenceAddFormContainer.js:33]: updated

**Review by rimselis — APPROVED** [review]

**Pull request merged**